packages/web-react/src/components/Link/Link.tsx

```diff
@@ -1,18 +1,22 @@
-import React, { ElementType, ForwardedRef, forwardRef } from 'react';
+import React, { ElementType, forwardRef } from 'react';
 import classNames from 'classnames';
 import { useStyleProps } from '../../hooks';
-import { SpiritLinkProps } from '../../types';
+import { SpiritLinkProps, PolymorphicRef } from '../../types';
 import { useLinkStyleProps } from './useLinkStyleProps';
 
 const defaultProps = {
   color: 'primary',
 };
 
+/**
```

Comment:
IMHO this should be near the type definition, where it is, so I suggest removing it from here. I think we will use this type in multiple places soon and copy of this comment will be unnecessary.

Reply:
removed